backend/usecase/article_usecase.go

```diff
@@ -0,0 +1,42 @@
+package usecase
+
+import "github.com/junichi-fukushima/tech-flow/backend/domain/article"
+
+type ArticleUsecase interface {
+	GetArticles() error
+	UpsertArticles() error
+}
+
+type articleUsecase struct {
+	articleRepository article.ArticleRepository
+}
+
+func NewArticleUsecase(articleRepository article.ArticleRepository) ArticleUsecase {
+	return &articleUsecase{articleRepository: articleRepository}
+}
+
+// 記事一括取得
+func (u *articleUsecase) GetArticles() error {
+	// feeds テーブルからデータを取得
+	_, err := u.articleRepository.GetArticlesAll()
+	// feed, err := u.articleRepository.GetFeeds()
+
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// 記事一括更新
+func (u *articleUsecase) UpsertArticles() error {
+	// feeds テーブルからデータを取得
+	_, err := u.articleRepository.GetArticlesAll()
+	// feed, err := u.articleRepository.GetFeeds()
+
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
他の部分もそうですが、下記の書き方と等価なので短くしちゃってもよいかもです
```suggestion
	return err
```